meta/saimetadatatypes.h

```diff
@@ -1019,6 +1068,11 @@ typedef struct _sai_attr_metadata_t
      */
     bool                                        isresourcetype;
 
+    /**
+     * @brief Determines whether attribute is MACSEC rule match field.
+     */
+    bool                                        ismacsecrulematchfield;
```

Comment:
need update perse.pl script tosupport that and you would need probably introduce a new tag? when this will be true ?

Reply:
Not relevant anymore as we changed from MACsec rule to ACL API.